dotCMS/src/integration-test/java/com/dotcms/contenttype/test/ContentResourceTest.java

```diff
@@ -1216,6 +1373,15 @@ private User createUser() throws DotSecurityException, DotDataException {
         return newUser;
     }
 
+    private CategoryDataGen createCategory(final String categoryName, final int sortOrder) {
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyyMMdd-HHmmss");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'simpleDateFormat' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4885353)